chart/instana-agent/Chart.yaml

```diff
@@ -1,24 +1,22 @@
-apiVersion: v1
-name: instana-agent
-version: 1.0.23
-appVersion: 1.7
+annotations:
+  artifacthub.io/links: |
+    - name: Instana website
+      url: https://www.instana.com
+    - name: Instana Helm charts
+      url: https://github.com/instana/helm-charts
+apiVersion: v2
+appVersion: 1.213.0
 description: Instana Agent for Kubernetes
 home: https://www.instana.com/
-icon: http://instana-management-assets.s3-website-eu-west-1.amazonaws.com/stan_icon_front_black_big.png
-sources:
-  - https://github.com/instana/instana-agent-docker
+icon: https://agents.instana.io/helm/stan-logo-2020.png
 maintainers:
-  - name: jbrisbin
-    email: jon.brisbin@instana.com
-  - name: wiggzz
-    email: william.james@instana.com
-  - name: JeroenSoeters
-    email: jeroen.soeters@instana.com
-  - name: fstab
-    email: fabian.staeber@instana.com
-  - name: mdonkers
-    email: miel.donkers@instana.com
-  - name: dlbock
-    email: dahlia.bock@instana.com
-  - name: nfisher
-    email: nathan.fisher@instana.com
+- email: henning.treu@ibm.com
+  name: htreu
+- email: m.donkers@ibm.com
```

Comment:
Might make sense to remove Miel here already (sorry about that) and include myself.